text/0004-channel-participation-api-without-system-channel.md

```diff
@@ -50,14 +50,21 @@ starts by scanning the system channel in order to discover existing channels and
 channels increase, this scan prolongs the process of adding a new orderer.  
 
 ## Operational problems
-There are several operational problems that affect large scale deployment and hinder efficient management of cloud
-environments using automatic tooling.
+The dependence on a system channel causes channel management to become unnecessarily fragile and complicated, impacting
+large scale deployment and hindering efficient management of cloud environments using automatic tooling.
+
+* It is not possible to spin up ordering service nodes without first coordinating on a system channel genesis block.
```

Comment:
, which in turn requires that the other bootstrap parameters of the ordering service nodes have already been generated.

Reply:
@jyellick I made the update, but I don't fully understand it... for my own education, what is an example of a bootstrap parameter that must be generated.

Reply:
For instance, it is typical model for deployment is that TLS certificates are generated at startup if they do not exist.  But, the orderer TLS cert is a parameter to the channel configuration consensus parameters.  So, with a system channel, we must first generate all of the orderer TLS artifacts before starting any orderer (because we need those artifacts to build the system channel).  With the channel participation API, we can let orderers start and come up fully before collecting those artifacts to build a channel configuration.